proposals/0031-hlsl-vector-matrix-operations.md

```diff
@@ -160,7 +160,61 @@ TBD
 
 ### Builtins
 
-TBD
+Although these "builtins" are not intended to be part of the HLSL language, and
+no promises are made that these will continue to be available over time, this
+proposal describes an implementation in terms of builtins such as this. For
+this reason it is useful to have them described here as a reference point.
+
+Each builtin corresponds to one of the operations described in [0029].
```

Comment:
The link to [0029] doesn't work.

Reply:
It works on my machine:

![image](https://github.com/user-attachments/assets/fc8aaa17-d8ac-4d23-9624-60d4fd96d674)